index.html

```diff
@@ -1,236 +1,221 @@
-<!DOCTYPE HTML>
+<!DOCTYPE html>
 <html>
 <head>
     <title>Project Rainbow</title>
-    <meta charset="utf-8"/>
-    <meta name="viewport" content="width=device-width, initial-scale=1"/>
-    <!--[if lte IE 8]>
-    <script src="assets/js/ie/html5shiv.js"></script><![endif]-->
-    <link rel="stylesheet" href="assets/css/main.css"/>
-    <link rel="stylesheet" href="assets/css/rainbow.css"/>
-    <!--[if lte IE 8]>
-    <link rel="stylesheet" href="assets/css/ie8.css"/><![endif]-->
-    <!--[if lte IE 9]>
-    <link rel="stylesheet" href="assets/css/ie9.css"/><![endif]-->
+    <meta charset="UTF-8">
+    <meta content="width=device-width, initial-scale=1" name="viewport">
+    <link href="http://www.w3schools.com/lib/w3.css" rel="stylesheet">
+    <link href="http://www.w3schools.com/lib/w3-theme-teal.css" rel="stylesheet">
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.3.0/css/font-awesome.min.css" rel="stylesheet">
+    <link href="assets\css\rainbow.css" rel="stylesheet">
 </head>
 <body>
 
-<!-- Nav -->
-<nav id="nav">
-    <ul class="container">
-        <li class="logo"><img src="images/logo.png" style='height: auto; max-height: 45px'></li>
-        <li><a href="#starting">Get Rainbow</a></li>
-        <li><a href="#plugins">Plugins</a></li>
-        <li><a href="#contribute">Contribute</a></li>
-        <li><a href="http://project-rainbow.org/site/">Forums</a></li>
+    <!-- Side Navigation -->
+    <ul class="w3-navbar w3-theme-2">
+        <li>
+            <a class="w3-theme-blue w3-padding-8" href="javascript:void(0)">Home</a>
+        </li>
+        <li>
+            <a class="w3-padding-8" href="#starting">Get Rainbow</a>
+        </li>
+        <li class="w3-dropdown-hover">
+            <a class="w3-padding-8" href="http://www.project-rainbow.org/site/index.php?action=downloads">Plugins&nbsp;<i class=
+            "fa fa-caret-down"></i></a>
+            <div class="w3-dropdown-content w3-card-4">
+                <a class="w3-padding-8" href="http://www.project-rainbow.org/site/index.php?action=downloads">Offical List</a>
+                <a class="w3-padding-8" href="https://isaiahpatton.github.io/RainbowPluginList">Fanmade list</a>
```

Comment:
Good idea. But clicking the buttons or _More info_ links doesn't seem to work for me (using Firefox).